zebra-network/src/protocol/external/types.rs

```diff
@@ -106,6 +106,9 @@ impl Version {
             (Mainnet, Nu5) => 170_100,
             (Testnet(params), Nu6) if params.is_default_testnet() => 170_110,
             (Mainnet, Nu6) => 170_120,
+            // FIXME: use proper values for Nu7
+            (Testnet(params), Nu7) if params.is_default_testnet() => 170_111,
+            (Mainnet, Nu7) => 170_121,
```

Comment:
The only requirement here is that there should be some versions between the prior min specified protocol versions and new ones in case something goes wrong with the prior NU deployment.

This PR could also update the `CURRENT_NETWORK_PROTOCOL_VERSION` (and should if it includes an NU7 activation height on Testnet, though it would be better to remove the Testnet activation height).

```suggestion
            (Testnet(params), Nu7) if params.is_default_testnet() => 170_130,
            (Mainnet, Nu7) => 170_140,
```

Reply:
Link to this comment copied to #37

Reply:
Fixed - removed FIXME comment and put under #[cfg(zcash_unstable = "nu6" /* TODO nu7 */ )]